README.md

```diff
@@ -85,50 +136,53 @@ Bria enables transaction batching and UTXO management providing the liquidity of
   - mempool - self-hostable backend used for fee estimations
   - otel-agent: optional for observability
 
-* For testing start the preconfigured dependencies with docker
-    ```
-    docker compose up -d integration-deps
-    ```
-* provide the connection parameters as environment variables
-
-    ```
-    # create the .envrc file
-    cat <<EOF > .envrc
-    export PG_HOST=127.0.0.1
-    export PG_CON=postgres://user:password@${PG_HOST}:5432/pg
-    EOF
+* provide the database connection parameters as environment variables
+  ```
+  # create a .envrc file
+  cat <<EOF > .envrc
+  export PG_HOST=127.0.0.1
+  export PG_CON=postgres://user:password@${PG_HOST}:5432/pg
+  EOF
 
-    direnv allow
-    ```
+  direnv allow
+  ```
 
 ## Usage
 
 * provide a database encryption key
   ```
   export SIGNER_ENCRYPTION_KEY="0000000000000000000000000000000000000000000000000000000000000000"
   ```
-
 * start the bria daemon with the [default configuration](tests/e2e/bria.local.yml)
   ```
-  bria daemon -c .tests/e2e/bria.local.yml run
+  bria daemon -c ./tests/e2e/bria.local.yml run
+  ```
+* create an admin API key (stored in the .bria folder)
+  ```
+  bria admin bootstrap
   ```
-* create an API key (stored in the .bria folder)
-bria admin bootstrap
-
 * list accounts
   ```
   bria admin list-accounts
   ```
+* create the profile API key (scoped to the account)
```

Comment:
This creates an account - the profile api key is created as a side effect. You can also explicitly create new profiles and new profile api keys. But an account must be created first.

Reply:
corrected